beesdoo_shift/models/planning.py

```diff
@@ -178,3 +178,28 @@ def _generate_task_day(self):
                 })
 
         return tasks
+
+    @api.onchange('worker_ids')
+    def check_for_multiple_shifts(self):
+        original_ids = {worker.id for worker in self._origin.worker_ids}
+
+        warnings = []
+        for worker in self.worker_ids:
+            if worker.id not in original_ids:
+                shift_ids = {shift.id for shift in worker.subscribed_shift_ids if shift.id != self.id}
```

Comment:
Need to test with a shift.template that does not exist in database yet.
Click on create do not save and add some workers that are already assigned to other shift

Reply:
shift_ids = {shift  for shift in worker.subscribed_shift_ids if shift.id != self.id}